tetraelf-hierarchy/hierarchy-api/api/src/java/org/sakaiproject/hierarchy/api/PortalHierarchyService.java

```diff
@@ -211,6 +211,13 @@
 	 * @return <code>true</code> if the node can have it's site changed.
 	 */
 	boolean canChangeSite(String id);
-	
-	
+
+	/**
```

Comment:
The API is supposed to be the services that you expose to the reset of Sakai, it's your contract to other tools/services wanting to use your functionality. Having a proxy method as part of that API leads to the API being larger and so harder to understand. Also someone reading the code who has seen `SiteService.siteExists()` can understand it somewhere else, however if I see `PortalHierarchService.siteExists()` I need to read JavaDoc/source to check what it actually does.